pkg/helpers/pointers.go

```diff
@@ -1,9 +1,9 @@
 package helpers
 
 func IsStringPtrNilOrEmtpy(ptr *string) bool {
-	if ptr == nil || *ptr == "" {
-		return true
-	}
+	return ptr == nil || *ptr == ""
+}
 
-	return false
+func IsBoolPtrTrue(ptr *bool) bool {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported function IsBoolPtrTrue should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=5024349)